content/Posts/implicit-permissions-of-a-service-account-in-kubernetes/index.md

```
@@ -0,0 +1,169 @@
+---
+
+Service accounts in Kubernetes are the identity of the workload.
+For every namespace, Kubernetes creates a service account by default and any pod that does not have an explicit service account assigned to it will run as the default service account for that namespace.
+
+![](./image.png)
+
+In this blog post we will see what permissions a default service account gets by default.
+Let's start by creating a fresh namespace so we can clean up everything at the end.
+
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)